### ISSUE TITLE
Give option to specify -PackagesDirectory or -SolutionDirectory

### ISSUE THREAD

**Issue by cable729** [tparnell8/gulp-nuget-restore#1]
Hi, thanks for the great gulp package. I've used it for restoring solution nuget packages, but now that I'm trying to create a more focused fast build, I'm trying to restore just a few packages from a .csproj. This is what happens:

```
MSBuild auto-detection: using msbuild version '4.0' from 'C:\Windows\Microsoft.NET\Framework\v4.0.30319'.

Cannot determine the packages folder to restore NuGet packages. Please specify either -PackagesDirectory or -SolutionDirectory.

C:\app\node_modules\gulp-nuget-restore\index.js:32
                throw error;
                ^

Error: Command failed: C:\app\node_modules\gulp-nuget-restore./nuget.exe restore C:\app\UnitTests\TestUtils\TestUtils\TestUtils.csproj
Cannot determine the packages folder to restore NuGet packages. Please specify either -PackagesDirectory or -SolutionDirectory.
```

I'm happy to give this a shot on my own if you don't have time. I haven't worked on npm packages before, though.

**Comment by TerribleDev** [comment]
v0.3.0 adds additional arguments where you can specify things such as packagedirectory

**Comment by cable729** [comment]
:+1: Thanks for the fast support!

**Comment by cable729** [comment]
Hey @tparnell8 -- Looks like the 0.3.0 package on npmjs.com is missing `index.js`.

**Comment by TerribleDev** [comment]
@cable729 weird..i'll take a peek!

**Comment by TerribleDev** [comment]
@cable729 not sure how that happened. Just published 0.3.1 and I downloaded it right after (it contains index.js)

**Comment by cable729** [comment]
Awesome! Yeah, it was strange. It was included in your release `.zip`. Thanks for the fix
